lib/ipv6.js

```diff
@@ -321,7 +315,7 @@ Address6.prototype.possibleSubnets = function (optionalSubnetSize) {
  */
 Address6.prototype._startAddress = function () {
   return new BigInteger(
-    this.mask() + repeat('0', constants6.BITS - this.subnetMask), 2
+    this.mask() + '0'.repeat(constants6.BITS - this.subnetMask), 2
```

Comment:
`string.repeat()` is not supported in Android Browser (via caniuse.com) but I don't have a clear sense of whether we care (recent release date but 0% usage)

Reply:
As long as this change goes out as a major version increment you'll not break existing users. If you get bug reports you can easily add it back in a patch release.